### PR TITLE
Improve Performance logging

### PR DESCRIPTION
**Pull request by bz888**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (#1233 )

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have tested locally
- [x] I have performed a self review of my changes
- [x] Linked to any relevant issues
- [x] I have added tests relevant to my changes
- [x] My code is up to date with the base branch

### CODE REVIEW

File: packages/node/src/indexer/store.service.ts [hunk 1]
```diff
@@ -110,6 +110,12 @@ export class StoreService {
     }
   }
 
+  async incrementBlockCount(key: string): Promise<void> {
```
Comment (stwiname) [inline packages/node/src/indexer/store.service.ts:113]: This should us a db transaction and I think the key can be hard coded

File: packages/node/src/indexer/worker/block-dispatcher.service.ts [hunk 2]
```diff
@@ -212,17 +212,20 @@ export class BlockDispatcherService
         blockNums,
       );
 
+      const processedBlockCount =
```
Comment (stwiname) [inline packages/node/src/indexer/worker/block-dispatcher.service.ts:215]: Getting this every time doesn't make much sense. I think we can keep track of it in memory.

Comment (jiqiang90) [reply packages/node/src/indexer/worker/block-dispatcher.service.ts:215]: Agree, we should only get this once when project initialized

**Comment by stwiname** [comment]
I think we should log the processed bps along with he current bps

File: packages/node/src/indexer/store.service.ts [hunk 3]
```diff
@@ -110,6 +110,12 @@ export class StoreService {
     }
   }
 
+  async incrementBlockCount(): Promise<void> {
```
Comment (stwiname) [inline packages/node/src/indexer/store.service.ts:113]: This should use the db transaction

**Review by stwiname — APPROVED** [review]

**Review by jiqiang90 — APPROVED** [review]

**Pull request merged**